site/_js/utils/events.js

```diff
@@ -0,0 +1,14 @@
+const startOfDay = new Date();
+startOfDay.setHours(0, 0, 0, 0);
+
+const isPastEvent = event => {
+  return new Date(event.date).getTime() < startOfDay.getTime();
+};
+
+const sortDescByTime = (a, b) =>
+  new Date(b.date).getTime() - new Date(a.date).getTime();
```

Comment:
EleventyCollectionItem.date already is a `Date` AFAIK, so there wouldn't be a need to recast it as a date in those sorting functions. Or are you sorting anything else with those?

Reply:
So this is used again on the frontend after the events have been serialized to JSON, at which point they are strings. It's not strictly necessary at this point as they'll be in the right order in the dump anyway. However, there is an outstanding requirement to dynamically move elapsed events over to the past events tab at which point this will be necessary.

Reply:
Ah I see. Now I also remember having stumbled over this before... I personally not a fan of mixing "backend" (11ty) code with frontend code but also can not say if that is already the case anywhere else in the codebase. @devnook, WDYT?

Reply:
Yep, I like that!